### PR TITLE
Lower deck panel support clips 

### PR DESCRIPTION
**Pull request by wile-e1**
There are some deck panel support clips I designed quite a while ago. I found that my coroplast panel for my V2.1 was a little too flimsy and could use a little more support.  I just updated the lower mount to be adjustable as well so that it can accomodate deck panels from 2mm to 6mm.

### CODE REVIEW

File: printer_mods/wile-e1/Deck_Panel_Support_Clips/readme.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+### Deck Panel Support Clips
+
+These are simple panel support clips to help better support the lower deck panels of a V2.4 and other similar models.  They sandwich the deck sheet between an upper and lower support and are held in with M3 SHCS or M3 BHCS.
+
+<img src="G:\Dropbox\github\VoronUsers\printer_mods\wile-e1\Deck_Panel_Support_Clips\deck_clip_top.png" alt="deck_clip_top" style="zoom:80%;" />
```
Comment (KiloQubit) [inline printer_mods/wile-e1/Deck_Panel_Support_Clips/readme.md:5]: These image references are broken, please update them to reference the images you uploaded with your mod, e.g.

    deck_clip_top

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!

- Please remove the almost empty directory `printer_mods/wile-e1/Mods/` and its empty readme
- Did you mean to have the clips print on their side instead of their bottom?
- deck panel support upper has an unprintable overhang at the bottom of the hole

Screen Shot 2021-06-17 at 20 54 53

File: printer_mods/wile-e1/Deck_Panel_Support_Clips/readme.md (hunk 1 above)
Comment (wile-e1) [reply printer_mods/wile-e1/Deck_Panel_Support_Clips/readme.md]: Image links in readme.md are now repaired.

**Review by KiloQubit — APPROVED** [review]
Thanks for the updates!

**Pull request merged**